modules/highgui/src/window_QT.cpp

```diff
@@ -903,6 +903,16 @@ double GuiReceiver::getPropWindow(QString name)
     return (double) w->getPropWindow();
 }
 
+double GuiReceiver::getWindowVisible(QString name)
```

Comment:
I have a build error here because the method `getWindowVisible` is not defined in `window_QT.h`.